src/parsers/trades/pionex/index.ts

```diff
@@ -0,0 +1,74 @@
+import { getCSVData } from '../../';
+import { EXCHANGES, IImport, IPartialTrade, ITrade } from '../../../types';
+import { createDateAsUTC, createID } from '../../utils';
+
+interface IPionex {
+    'date(UTC+0)': string;
+    amount: string;
+    price: string;
+    order_price: string;
+    side: string;
+    symbol: string;
+    state: string;
+    fee: string;
+    strategy_type: string;
+}
+
+export async function pionexParser(importDetails: IImport): Promise<ITrade[]> {
+    const data: IPionex[] = await getCSVData(importDetails.data) as IPionex[];
+    const internalFormat: ITrade[] = [];
+    for (const trade of data) {
+        const tradeToAdd: IPartialTrade = {};
+        tradeToAdd.date = createDateAsUTC(new Date(trade['date(UTC+0)'])).getTime();
+        tradeToAdd.exchange = EXCHANGES.Pionex;
+        if (trade.side === 'BUY') {
+            tradeToAdd.boughtCurrency = trade.symbol.substr(0,trade.symbol.lastIndexOf('_'));
```

Comment:
Can there be multiple `_` in a symbol? If not we should be able to just use a split here
```
const [boughtCurrency, soldCurrency] =  trade.symbol.split('_')
```

Reply:
There is only one, so yes we could

Reply:
Done, please check